pep/app/pods/components/page/sidebar/widgets/template.hbs

```diff
@@ -1,15 +1,18 @@
-<div class="p-3 d-flex">
-    <a href="" {{on "click" this.toggleAll}}>
+<div class="p-3 d-flex justify-content-between align-items-center">
+    <h5 class="m-0">{{t "relatedInfo.title"}}</h5>
+    <Button @type="link" class="p-0" {{on "click" this.toggleAll}}>
         {{t (if this.showCloseAll 'relatedInfo.batchClose' 'relatedInfo.batchOpen')}}
-    </a>
+    </Button>
 </div>
```

Comment:
probably should have `px-2 py-3` instead of `p-3` for this container to match the widgets' horizontal padding. 

use `@action={{this.toggleAll}}` instead of `{{on "click" this.toggleAll}}` on `<Button>`'s.